packages/roosterjs-editor-api/lib/index.ts

```diff
@@ -38,3 +37,7 @@ export { default as toggleSubscript } from './format/toggleSubscript';
 export { default as toggleSuperscript } from './format/toggleSuperscript';
 export { default as toggleUnderline } from './format/toggleUnderline';
 export { default as toggleHeader } from './format/toggleHeader';
+
+// @deprecated the function getPendableFormatState will still be available from
+// roosterjs-editor-dom package, keep export it here just for compatibility
+export { getPendableFormatState } from 'roosterjs-editor-dom';

```

Comment:
Does this mean that roosterjs-editor-api now has a hard dependency on roosterjs-editor-dom?